pytorch_lightning/accelerators/ddp_base_backend.py

```diff
@@ -13,20 +13,21 @@
 # limitations under the License
 import os
 import re
-import torch
 
-from pytorch_lightning.utilities import AMPType
-from pytorch_lightning.accelerators.base_backend import Accelerator
+import torch
 import torch.distributed as torch_distrib
 import torch.distributed as dist
-from pytorch_lightning.utilities.cloud_io import atomic_save
-from pytorch_lightning.utilities.distributed import rank_zero_warn, rank_zero_only
+
 from pytorch_lightning import _logger as log
+from pytorch_lightning.accelerators.base_backend import Accelerator
```

Comment:
this is good but rather make it in separate PR

Reply:
this is just isort